src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala

```diff
@@ -84,6 +84,27 @@ class ClaimSimulation extends Simulation {
     })
   )
 
+  val randomNhsNos = Iterator.continually(
+    // Random number will be accessible in session under variable "OrderRef"
```

Comment:
Comment mentions OrderRef, also won't be in session this function is just creating the nhsno's

Reply:
changed accordingly